### ISSUE TITLE
On Windows the menu bar isn't detected by screen readers.

### ISSUE THREAD

**Issue by OzancanKaratas** [deltachat/deltachat-desktop#1102]
In the Windows builds, menu bar is not displayed. I have tried following to reach the menu bar:

- Pressed F10: Not displayed.
- Pressed Alt key: Not displayed.
- Pressed Shift+F10: Not displayed.

Operating system: Windows 10 version 19H1.

Delta Chat build (this operations tried in this build environment): Both portable and installed versions.

**Comment by Simon-Laux** [comment]
can't reproduce on windows 10

**Comment by Simon-Laux** [comment]
Is this still happening?

**Comment by OzancanKaratas** [comment]
Yes.

**Comment by Simon-Laux** [comment]
can you somehow send us a screenshot of this?
as it might contain sensitive data please send it to delta@merlinux.eu instead of this public github thread.

**Comment by OzancanKaratas** [comment]
No need because I noticed the menu bar is displayed. But not in a right place. So the screen readers does not detect it. I scanned the screen using OCR.

**Comment by Jikstra** [comment]
I don't really know how to reproduce this. Is it possible that the menu color is black on black on windows? @OzancanKaratas is it possible for you to send a screenshot of the window to jikstra*AT*disroot.org ?

**Comment by OzancanKaratas** [comment]
@Jikstra

This issue can be fixed by adding the Alt key as a hotkey to open the menu bar.

**Comment by OzancanKaratas** [comment]
Hi,

No problem in latest version. How did you fixed it?

**Comment by Simon-Laux** [comment]
we recently updated the electron version, maybe that fixed it.

**Comment by Simon-Laux** [comment]
I will close this, please reopen if it appears again. I assume this was an electron bug.
